leaderBoard/index.js

```diff
@@ -0,0 +1,221 @@
+let userData;
+const input = document.getElementsByName("search");
+
+let solutions = [];
+let metrics = [];
+let winners = [];
+
+let postPoint = 1;
+let likePoint = 2;
+let solutionPoint = 3;
+
+
+let baseURL = "https://discuss.layer5.io/";
+let config = {
+  headers: {
+    "Api-Key":
```

Comment:
Remove the API keys. Should be private. I think the API query should work without the API key and username

Reply:
yes it should, i would remove them now